Makefile

```diff
@@ -389,6 +389,7 @@ bin_modules-$(CONFIG_FLASHTOOLS) += flashtools
 bin_modules-$(CONFIG_NEWT) += newt
 bin_modules-$(CONFIG_CAIRO) += cairo
 bin_modules-$(CONFIG_FBWHIPTAIL) += fbwhiptail
+bin_modules-$(CONFIG_NITROKEY) += nitrokey-hotp-verification
```

Comment:
Is there a separate PR coming for nitrokey support?

(This is a NOP since `CONFIG_NITROKEY` will not be yes, so it's ok for now)

Reply:
Yes, that was unfortunately copied over by mistake. The config options et al will be a bit different with the PR related to Librem Key support.